packages/devtools_app/lib/src/flutter/utils.dart

```diff
@@ -14,3 +15,31 @@ Future<void> launchUrl(String url, BuildContext context) async {
     Notifications.of(context).push('Unable to open $url.');
   }
 }
+
+List<TextSpan> processAnsiTerminalCodes(String input, TextStyle defaultStyle) {
+  if (input == null) {
+    return [];
+  }
+  return decodeAnsiColorEscapeCodes(input, AnsiUp())
+      .map((entry) => TextSpan(
+            text: entry.text,
+            style: entry.style.isEmpty
+                ? defaultStyle
+                : TextStyle(
+                    color: entry.fgColor != null
+                        ? colorFromAnsi(entry.fgColor)
+                        : null,
+                    backgroundColor: entry.bgColor != null
+                        ? colorFromAnsi(entry.bgColor)
+                        : null,
+                    fontWeight:
+                        entry.bold ? FontWeight.bold : FontWeight.normal,
+                  ),
+          ))
```

Comment:
nit: maybe add a trailing comma here to reduce the indent but this case is fine either way.